explorer/explorer.go

```diff
@@ -172,7 +179,19 @@ func NewExplorer(cm ChainManager, store Store, indexCfg config.Index, scanCfg co
 				e.log.Error("failed to get tip", zap.Error(err))
 			}
 			if err := e.syncStore(lastTip, indexCfg.BatchSize); err != nil {
-				e.log.Panic("failed to sync store", zap.Error(err))
+				switch {
+				case errors.Is(err, context.Canceled):
+					return
+				case strings.Contains(err.Error(), "missing block at index"):
```

Comment:
we should open a PR on coreutils to expose this error, it only takes a second and we're updating coreutils/all dependencies every day anyway

Reply:
I'd create a separate issue to track this. This shouldn't block this PR and we have multiple repos that need to be updated once we have it.

Reply:
https://github.com/SiaFoundation/coreutils/pull/217

Reply:
https://github.com/SiaFoundation/coreutils/pull/217